components/input_environmental_personal.py

```diff
@@ -337,22 +384,26 @@ def handle_modal(clo_value, _nc_open, _nc_close, _nc_submit, opened):
     ctx = dash.callback_context.triggered_id
 
     if ctx == ElementsIDs.modal_custom_ensemble_open.value:
-        return True, dash.no_update
+        return True, dash.no_update, "none", dash.no_update
 
     if ctx == ElementsIDs.modal_custom_ensemble_close.value:
-        return False, dash.no_update
+        return False, dash.no_update, "none", dash.no_update
 
     total_clo_value = 0
     for value in clo_value:
         total_clo_value += float(value.split("_")[0])
     total_clo_value = round(total_clo_value, 2)
 
-    if ctx in [
-        ElementsIDs.modal_custom_ensemble_submit.value,
-    ]:
-        return False, total_clo_value
+    if total_clo_value > 1.5:
```

Comment:
add a todo here since we will need to update this value based on the model max value, different models have different max values for clo

Reply:
ok

Reply:
Its now determine clo max and min value base on the models